src/Compilers/CSharp/Test/Syntax/Parsing/StatementParsingTests.cs

```diff
@@ -2597,7 +2597,7 @@ class C1
 {
     static void Test(int arg1, (byte, byte) arg2)
     {
-        (int, int) t1 = new(int, int)();
+        (int, int)? t1 = new(int, int)?();
```

Comment:
Can you please modify the test to assert what the diagnostics are? This syntax should also not work (i.e. a nullable tuple type syntax with `new` should also be a syntax error).